src/renderer/components/general-settings/general-settings.js

```diff
@@ -150,6 +155,18 @@ export default Vue.extend({
         this.$t('Settings.General Settings.Thumbnail Preference.Middle'),
         this.$t('Settings.General Settings.Thumbnail Preference.End')
       ]
+    },
+
+    externalLinkHandling: function () {
+      return this.$store.getters.getExternalLinkHandling
+    },
+
+    externalLinkHandlingNames: function () {
+      return [
+        this.$t('Settings.General Settings.External Link Handling.Open Link'),
+        this.$t('Settings.General Settings.External Link Handling.Open Link After Prompt'),
+        this.$t('Settings.General Settings.External Link Handling.Do Nothing')
```

Comment:
Bit of nitpick but I think "Don't Open Link" might be better text than "Do Nothing" (especially when considering translators might be missing the context of the string)

Reply:
I found it strange to read "not doing specific thing" as an "action to be performed by the application".
I can consider changing it to "No Action" if that sounds better to you.

Reply:
No Action would be better

Reply:
Updated to become "No Action"